src/search_bar/styles.tsx

```diff
@@ -0,0 +1,42 @@
+import { makeStyles } from '@material-ui/styles';
+import { useTheme } from '@material-ui/core/styles';
+
+export const useGetStyles = () => {
+  const theme: any = useTheme();
+
+  const input = {
+    background: theme?.palette?.background?.paper
+    ?? 'rgba(0, 0, 0, 1)',
+  };
+
+  const icon = {
+    color: theme?.palette?.custom?.fonts?.fontFour ?? 'rgba(0, 0, 0, 1)',
+  };
+
+  const useStyles = makeStyles({
+    root: {
```

Comment:
the height of search bar should be 2.5rem(40px),  but it's 36px now. And the height of search bar topBar(36px now) and headerBar(43px now) should be the same.